### DISCUSSION TITLE
How do I delete a blocked access repository?

### DISCUSSION THREAD

**Discussion by PR3C14D0** [community/community#31278] [Other Feature Feedback, Questions, & Ideas]
So, I want to delete one of my blocked access repository from the API but I can't. How can I delete?

![image](https://user-images.githubusercontent.com/60857589/187313626-c01a35ab-1308-419e-b36b-6491db978e7e.png)
Thx.

**Comment by airtower-luna** [comment]
That's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Good luck!

**Comment by PR3C14D0** [comment]
Right, thanks ^-^
